portfolio/src/main/webapp/gallery.css

```diff
@@ -0,0 +1,69 @@
+* {box-sizing: border-box;}
+body {font-family: Verdana, sans-serif;}
+.mySlides {display: none;}
+img {vertical-align: middle;}
+
+/* Slideshow container */
+.slideshow-container {
+  max-width: 1000px;
+  position: relative;
+  margin: auto;
+}
+
+/* Caption text */
+.text {
+  color: #f2f2f2;
+  font-size: 15px;
+  padding: 8px 12px;
+  position: absolute;
+  bottom: 8px;
+  width: 100%;
+  text-align: center;
+}
+
+/* Number text (1/3 etc) */
+.numbertext {
+  color: #f2f2f2;
```

Comment:
this color can be a constant

Reply:
I tried to use find syntax on how to create a color constant in css, but I couldn't find any syntax.

Reply:
usually frontend framework will have someting to faciiliate that. Since this is pure css, I think something like this https://stackoverflow.com/questions/4847850/is-it-possible-to-define-constants-in-css is fine.